internal/core/application/mocks_test.go

```diff
@@ -1,648 +0,0 @@
-package application_test
```

Comment:
I didin't know about this tool! Thanks for sharing :)

Reply:
I played with the tool a little bit and my preferred solution is to create a mocks directory in the folder layer (`domain` in this case), then import the mocks from there when testing. The resulting command (ran after cd-ing into internal/core/domain) is: `mockery --name=SwapParser --recursive --structname=MockSwapParser --filename=swap.go`. The filename should be the same of the one containing the interface being mocked.

Reply:
That sounds ok.
Maybe u can add make cmd for generating mocks like i did in [here](https://github.com/XDEX-SA/xdex-price-alerts/blob/b7da215d3d2bb36e610a62d8198828c2a54c5dae/Makefile#L99) since interface can change and then u can easily recreate all mocks with one command.

Reply:
Done.